docs/paper/admin/reference/system-properties.md

```diff
@@ -0,0 +1,183 @@
+- **description**: Enables explicit flushing of the network channel.
+
+#### Paper.enable-sync-chunk-writes:
+
+- **default**: `unset`
+- **description**: Syncs writes on each write call. This has a performance impact, particularly on hard drives.
+
+#### paper.debug-sync-loads:
+
+- **default**: `unset`
+- **description**: Enables debug logging for sync chunk loads.
+
+#### Paper.ignoreWorldDataVersion:
+
+- **default**: `unset`
+- **description**: Ignores the world data version when loading a world. This is not recommended and will likely cause issues.
+
+#### debug.entities:
+
+- **default**: `unset`
+- **description**: Enables debug log messages for entity information.
+
+#### Paper.bypassHostCheck:
+
+- **default**: `unset`
+- **description**: Bypasses the host pattern matching attempt for the client when connecting to the server.
+
+#### paper.ticklist-warn-on-excessive-delay:
+
+- **default**: `unset`
+- **description**: Enables the warning when a tick list is scheduled with an excessive delay.
+
+#### debug.rewriteForIde:
+
+- **default**: `unset`
+- **description**: Removes the NMS revision from the stack trace to allow for easier debugging in IDEs. 
+It also remaps plugin CB calls to remove the version information.
+
+#### convertLegacySigns:
+
+- **default**: `unset`
+- **description**: Converts legacy signs to the new format.
+
+#### paper.maxCustomChannelName:
+
+- **default**: `64`
+- **description**: Sets the largest size that a plugin channel can be.
+
+#### Paper.maxSignLength:
+
+- **default**: `80`
+- **description**: Sets the maximum line length for signs.
+
+#### Paper.FilterThreshhold:
+
+- **default**: `8192`
+- **min**: `8192`
+- **description**: Sets the maximum size of a packet to be filtered.
+
+#### Paper.minPrecachedDatafixVersion:
+
+- **default**: `Minecraft world version + 1`
+- **description**: If you are expecting to convert a large number of chunks you might consider setting this to only convert from a point onwards.
+
+#### Paper.WorkerThreadCount:
+
+- **default**: `8` or `number of cpus - 2`. Whichever is lower
+- **description**: Sets the number of worker threads to use for chunk loading.
+
+#### Paper.excessiveTELimit:
+
+- **default**: `750`
+- **description**: Splits tile entities into multiple packets if there are more than this many.
+
+#### io.papermc.paper.suppress.sout.nags:
+
+- **default**: `unset`
+- **description**: Suppresses the nag message about using System.out/System.err in a plugin.
+
+#### paper.strict-thread-checks:
+
+- **default**: `unset`
+- **description**: Overrides the status of the AsyncCatcher so that it will always throw an exception if it is not on the main thread.
+
+#### Paper.skipServerPropertiesComments:
+
+- **default**: `unset`
+- **description**: Skips the comments in the server.properties file.
+
+#### Paper.debugInvalidSkullProfiles:
+
+- **default**: `unset`
+- **description**: Enables debug logging for invalid skull profiles. It appends an exception to the log.
```

Comment:
Instead of appending exceptions, say that it "This logs any invalid skulls in the world with location information to help properly track them down.